bazel/integration/tests/package_mappings/package.json

```diff
@@ -5,5 +5,10 @@
   "license": "MIT",
   "dependencies": {
     "fake_pkg": "0.0.0"
+  },
+  "resolutions": {
+    "**/fake_pkg": "0.0.0",
+    "some_pkg/**/fake_pkg": "0.0.0",
+    "fake_pkg": "0.0.0"
   }
```

Comment:
Could we also use `devDependencies` and `optionalDependencies` here?

Reply:
👍